examples/chef/create_docker.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import optparse
+import tarfile
+import shutil
+import docker
+
+client = docker.from_env()
+
+_CREATE_DOCKER_SCRIPT_PATH = os.path.dirname(__file__)
+_SUPPORTED_PLATFORM = 'linux_x86'
+
+parser = optparse.OptionParser()
```

Comment:
https://docs.python.org/3/library/optparse.html

```
Deprecated since version 3.2: The optparse module is deprecated and will not be developed further; development will continue with the argparse module.
```

lets not use deprecated libraries in new code.

Reply:
Thanks for pointing this out. I changed it to use argparse